scripts/prerelease/generate_prerelease_script.py

```diff
@@ -77,6 +78,12 @@ def main(argv=sys.argv[1:]):
     build_files = get_source_build_files(config, args.rosdistro_name)
     build_file = build_files[args.source_build_name]
 
+    # Use targets defined by corresponding Release config instead of
+    # targets defined by Source config. This is to increase the number of
+    # supported targets. New targets are explicitly passed to configure_devel_job().
+    release_build_files = get_release_build_files(config, args.rosdistro_name)
+    release_build_file = release_build_files[args.source_build_name]
```

Comment:
@dirk-thomas Thanks for clarifying the situation. How about something along this line to merge all the targets:

```python
    release_build_files = get_release_build_files(config, args.rosdistro_name)
    release_targets_combined = {}
    release_targets_combined[args.os_name] = {}
    for arch, rel_obj in release_build_files.items():
        for os, t in rel_obj.targets[args.os_name].items():
          if os not in release_targets_combined[args.os_name]:
            release_targets_combined[args.os_name][os] = {}
          release_targets_combined[args.os_name][os].update(t)

  # ...
  # pass release_targets_combined to configure_devel_job()
```